### DISCUSSION TITLE
Should we use WordPress Versioning?

### DISCUSSION THREAD

**Discussion by Rumperuu** [markcheret/footnotes#92] [Ideas]
Whilst we're in the process of refactoring the development and release process, it might be worth considering shifting from the use of Semantic Versioning (SemVer) to [WordPress Versioning](https://make.wordpress.org/core/handbook/about/release-cycle/version-numbering/). This would bring us in line with the rest of the ecosystem (i.e., some large Plugins like [WooCommerce](https://developer.woocommerce.com/2021/01/22/developer-advisory-changes-to-woocommerce-versioning-scheme/) also follow this style of versioning), but it's not a hard requirement.

**Comment by lolzim** [comment]
I agree, it should not break anything except a certain idea of versioning.

**Comment by markcheret** [comment]
I'm up for the challenge ;)
